recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
Thanks for your work on conda/conda-build#915 @jschueller.  I think @frol's idea of ensuring that MAKEFLAGS is passed appropriately requires less implementation.  I think it's just a matter of exporting it beforehand - I think conda build passes the existing shell's environment variables through.  Need to test this, though...